extensions/infinispan-client/runtime/src/main/java/io/quarkus/infinispan/client/runtime/InfinispanClientProducer.java

```diff
@@ -218,7 +231,24 @@ private ConfigurationBuilder builderFromProperties(Properties properties) {
                 .entrySet()) {
             String cacheName = cache.getKey();
             InfinispanClientRuntimeConfig.RemoteCacheConfig remoteCacheConfig = cache.getValue();
-            if (remoteCacheConfig.configuration.isPresent()) {
+            if (remoteCacheConfig.configurationUri.isPresent()) {
```

Comment:
@wburns @Sanne would getting the file content be necessary to cover different use cases in different scenarios (kube involved)  instead of using the URI directly with the remoteCache configuration?
This might be a very silly question from my side, is just something I was wondering we might need to cover.
If not, then I will remove the file and let the configuration and the configurationURI.
From Infinispan 14 @Sanne we will be able to pass json and yaml format too.

Reply:
I don't think using json or yaml would improve much, it's still odd to see such things inlined in a single property.

But as I mentioned above, it's perhaps acceptable in this case since there could be value in setting it as an environment variable.

Doubting myself - is that even true? I'm expecting that cache creation+configuration is mostly valuable during development; I'm not sure if I'd be deploying the client app in Kubernetes at that stage?

Also - I suppose we can't infer the "right" configurations the application is expecting? I used to be able to do that for Hibernate OGM: we'd automatically configure the grid, without our users needing to know about the configuration syntax.

Reply:
The yaml and properties I'm mentioning is related to using a configuration file that will be either in json or yaml too by the way! :))))
Concerning the right configuration expecting users for a cache in the server, I don't think we can infer it for them right now easily. We can however from Infinispan 14 improve the devconsole to point to the cache creation wizard and generate a configuration file either in xml, json or yaml

You can give it a try in the console right now
 docker run -it -p 11222:11222 -e USER="admin" -e PASS="password" quay.io/infinispan/server:14.0.0.CR1

Reply:
@Sanne the kube use case is only valid for demos IMO, and even then providing a cache cr is a more production real use case than creating it from the properties, but for simplicity is clearly something easy to be done.
I will remove the file and let URI and pasted configuration as it is